lib/chainweb/p2p/block_hash.ex

```diff
@@ -3,29 +3,62 @@ defmodule Kadena.Chainweb.P2P.BlockHash do
     BlockHash endpoints implementation for P2P API.
   """
 
-  @endpoint "hash"
+  @endpoints [get: "hash", post: "hash/branch"]
```

Comment:
It's not necessary to do this. You can provide a `:path` to the Request module, and it will be appended to the `:endpoint`.
https://github.com/kommitters/kadena.ex/blob/46f0ec9a2a00009222f037157d1a94be16bd0c7e/lib/chainweb/request.ex#L106-L126